src/Tizen.NUI/src/public/WindowSystem/QuickPanelService.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Tizen.NUI
+{
+    namespace WindowSystem
+    {
+        namespace WsShell
```

Comment:
Is `WsShell` namespace necessary? How about "Tizen.NUI.WindowSystem" ?

Reply:
We have many type of API.
namespace WsShell supports tizen-ws-shell API set.
and later, we will add C# mapping APIs not releated tizen-ws-shell package, like efl-util package

Reply:
I changed namespace
Tizen.NUI.WindowSystem.Shell
 class TizenShell, QuickPanelClient, QuickPanelService